### ISSUE TITLE
[Bug] Viewer role can pause alert

### ISSUE THREAD

**Issue by gaoyifan** [grafana/grafana#9640]
Please include this information:
- What Grafana version are you using?
 Grafana v4.5.2 (commit: ec2b0fe) 
- What datasource are you using?
influxdb
- What OS are you running grafana on?
official docker image
- What did you do?
Anonymous user (role=**Viewer**) visit https://grafana.example.com/alerting/list, click *pause* button.
- What was the expected result?
*Pause*(or *unpause*) action be denied.
- What happened instead?
alert was paused.

grafana.ini
```ini
[paths]
[server]
domain = ????
root_url = https://%(domain)s/
[database]
[session]
[analytics]
reporting_enabled = false
[security]
admin_password = ????
[snapshots]
[users]
allow_sign_up = false
[auth]
[auth.anonymous]
enabled = true
org_name = ????
[auth.github]
[auth.google]
[auth.generic_oauth]
[auth.grafananet]
[auth.proxy]
[auth.basic]
[auth.ldap]
enabled = true
[smtp]
enabled = true
host = ????:25
skip_verify = true
from_address = ????
[emails]
welcome_email_on_sign_up = true
[log]
filters = ldap:debug
[log.console]
[log.file]
[log.syslog]
[event_publisher]
[dashboards.json]
[alerting]
[metrics]
[metrics.graphite]
[grafana_net]
[external_image_storage]
[external_image_storage.s3]
[external_image_storage.webdav]
```

<img width="870" alt="image" src="https://user-images.githubusercontent.com/5450996/31933113-bcf6e5a8-b8da-11e7-9a99-8c7f0cafcdd5.png">

**Comment by bergquist** [comment]
Thank you for reporting this. Will be available in v4.6.0-beta4

**Comment by gaoyifan** [comment]
Thanks for your outstanding work!
